src/quo2/components/navigation/bottom_nav_tab/view.cljs

```diff
@@ -77,19 +81,10 @@
         (if (= notification-indicator :counter)
           [counter/counter
            {:override-text-color colors/white
-            :override-bg-color   colors/primary-50
-            :style               {:position :absolute
-                                  :left     48
-                                  :top      2}}
+            :override-bg-color   (colors/custom-color customization-color 60)
```

Comment:
maybe outside the scope, but perhaps good to create a follow up issue to track -> 
counter needs to use `customization-color` prop instead of override-bg-color. It should just be passed through, and in that component it can call colors/custom-color itself

Reply:
Yes, I think it should be a new issue, I've just created it: #16261